english/breakfast/breakfast.md

```diff
@@ -1,4 +1,4 @@
 You see a pot of coffee near you and you realize that you are hungry. Well, you poured coffee in a mug and picked some 
 marshmallows from the wall and added them to the coffee. While you are drinking the coffee and eating those delicious 
 marshmallows and staring at the wall you found that you made a whole in the wall by picking marshmallows. You finished
-your flavoured coffee and got out from marshmallows-walled room.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.